src/ui/view/dataverse/component/pageTemplate/pageTemplate.tsx

```diff
@@ -19,17 +23,26 @@ const isGeneralMetadata = (metadata: Metadata): metadata is GeneralMetadata =>
   metadata.property !== 'tags'
 
 const PageTemplate: FC<PageTemplateProps> = ({ data, metadata }): JSX.Element => {
+  const { t } = useTranslation('common')
+  const navigate = useNavigate()
+  const backToDataverse = useCallback((): void => navigate('/dataverse'), [navigate])
+
   const tags = pipe(
     metadata,
     A.filter(isTagsMetadata),
     A.chain(metadata => metadata.value)
   )
-
   const generalMetadata = pipe(metadata, A.filter(isGeneralMetadata))
 
   return (
     <div className="okp4-dataverse-portal-dataverse-component-page-template-main">
       <div className="okp4-dataverse-portal-dataverse-page-template-left-side-wrapper">
+        <div className="okp4-dataverse-portal-dataverse-back">
```

Comment:
back doesn't tell us much about the element. Seeing as this a container for the back button I'd call it that

```suggestion
        <div className="okp4-dataverse-portal-dataverse-back-button-container">
```

Reply:
good point!

Reply:
Yes you're right. I would keep it short though.